homeassistant/components/trello/strings.json

```diff
@@ -0,0 +1,33 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "api_key": "API Key",
+          "api_token": "Token"
```

Comment:
I think there are lokalize strings for these values, please check them and add them here